ovos_plugin_manager/templates/tts.py

```diff
@@ -44,272 +45,26 @@
-        while not self._terminated:
-            while self._paused:
-                sleep(0.2)
-            try:
-                # HACK: we do these check to account for direct usages of TTS.queue singletons
-                speech_data = self.queue.get(timeout=2)
-                if len(speech_data) == 5 and isinstance(speech_data[-1], Message):
-                    data, visemes, listen, tts_id, message = speech_data
-                else:
-                    LOG.warning("it seems you interfacing with TTS.queue directly, this is not recommended!\n"
-                                "new expected TTS.queue contents -> data, visemes, listen, tts_id, message")
-                    if len(speech_data) == 6:
-                        # old ovos backwards compat
-                        _, data, visemes, ident, listen, tts_id = speech_data
-                    elif len(speech_data) == 5:
-                        # mycroft style
-                        tts_id = None
-                        _, data, visemes, ident, listen = speech_data
-                    else:
-                        # old mycroft style  TODO can this be deprecated? its very very old
-                        listen = False
-                        tts_id = None
-                        _, data, visemes, ident = speech_data
-
-                    message = Message("speak", context={"session": {"session_id": ident}})
-
-                self._now_playing = (data, visemes, listen, tts_id, message)
-                self._play()
-            except Exception as e:
-                pass
-
-    def show_visemes(self, pairs):
-        """Send viseme data to enclosure
-
-        Args:
-            pairs (list): Visime and timing pair
-
-        Returns:
-            bool: True if button has been pressed.
-        """
-        if self.enclosure:
-            self.enclosure.mouth_viseme(time(), pairs)
-
-    def pause(self):
-        """pause thread"""
-        self._paused = True
-        if self.p:
-            self.p.terminate()
-
-    def resume(self):
-        """resume thread"""
-        if self._now_playing:
-            self._play()
-        self._paused = False
-
-    def clear(self):
-        """Clear all pending actions for the TTS playback thread."""
-        self.clear_queue()
-
-    def blink(self, rate=1.0):
-        """Blink mycroft's eyes"""
-        if self.enclosure and random.random() < rate:
-            self.enclosure.eyes_blink("b")
-
-    def stop(self):
-        """Stop thread"""
-        self._now_playing = None
-        self._terminated = True
-        self.clear_queue()
-
-    def shutdown(self):
-        self.stop()
-        for tts in self.attached_tts:
-            self.detach_tts(tts)
-
-    def __del__(self):
-        self.shutdown()
+            from ovos_audio.playback import PlaybackThread
+            return PlaybackThread(*args, **kwargs)
+        except ImportError:
+            raise ImportError("please install ovos-audio for playback handling")
```

Comment:
This import error is being thrown with ovos-audio 0.0.2a13 installed

Reply:
0.0.2a19 is the min version needed